checkbox-test-plans/README.md

```diff
@@ -1,19 +1,164 @@
-The following instructions were developed from setting up the checkbox testing tool on a lunar VM.
+This is a how-to guide on setting up checkbox on a 22.04 VM and executing manual tests with it. We ultimately would like to automate the checkbox test cases, with the use of other tools down the road.
 
-# Setting up checkbox dependencies
+Because a GitHub runner, running a workflow on ‘ubuntu-latest’, will be running 22.04. So I’m starting with a fresh 22.04 VM.
```

Comment:
Maybe.... "Because a GitHub runner, running a workflow on ‘ubuntu-latest’, will be running 22.04, we will start with a fresh 22.04 VM."

Reply:
sounds good